images/linux/Ubuntu1804-Readme.md

```diff
@@ -149,6 +149,7 @@
 - Rust 1.62.1
 - Rustdoc 1.62.1
 - Rustup 1.25.1
+- Sccache 0.3.0
```

Comment:
I was not sure where sccache should be mentioned in the docs. It is popular to use in the Rust ecosystem as a rustc wrapper, but it also works with other compilers too so is relevant to more ecosystems than just Rust.